hotfixes.js

```diff
@@ -4,27 +4,42 @@ var hotfixes = {
   "Products.PloneHotfix20170117": {
     "required_for_plone": [
           ["3", "3.99.??"],
-          ["4", "4.99.??"],
-          ["5", "5.99.??"]
+          ["4", "4.3.11"],
```

Comment:
Are these bounds inclusive or exclusive? Because either way I think one of them is wrong 😉 

> This hotfix should be applied to the following versions of Plone:
>
> Plone 5.0.6 and any earlier 5.x version
> Plone 4.3.11 and any earlier 4.x version
> Any older version of Plone